client/app/components/packages/rwcds-form/show.hbs

```diff
@@ -0,0 +1,250 @@
+<div class="grid-x grid-margin-x">
+  <div class="cell large-8">
+
+    <section class="form-section">
+      <h1 class="no-margin header-large">
+        <span id="project-info" class="section-anchor"></span>
+        Reasonable Worst Case Development Scenario
+      </h1>
+
+      <h2 class="section-header header-xxlarge"
+        data-test-show="dcpProjectname">
+        {{@package.rwcdsForm.dcpProjectname}}
+      </h2>
+    </section>
+
+    <section class="form-section">
+      <h2 class="section-header">
+        <span id="" class="section-anchor"></span>
+        Description
+      </h2>
+
+      <p class="grid-x">
+        <strong class="cell auto large-padding-right">
+          Has the project changed since submission of the Pre-Application Statement (PAS)?
+        </strong>
+        <span class="cell large-3">
+          {{if @package.pasForm.dcpHasprojectchangedsincesubmissionofthepas "Yes" "No"}}
```

Comment:
2 things:
1) should say `@package.rwcdsForm.dcpHas...`
2) We should use the `{{optionset ...}}` helper here, so that we don't need to hardcode the labels.
```
{{#if @package.rwcdsForm.dcpHasprojectchangedsincesubmissionofthepas}}
   {{optionset 'rwcdsForm' 'dcpHasprojectchangedsincesubmissionofthepas' 'label' @package.rwcdsForm.dcpHasprojectchangedsincesubmissionofthepas}}
{{/if}}
```

I know it seems verbose, but programmatically it is actually cleaner

Reply:
Ack! Thanks for catching that copy/paste "pas".

Reply:
Why do we need the if here?

Reply:
The code you provide will work for "YES" but not for "NO".